lib/brains/forms.py

```diff
@@ -85,3 +85,23 @@ def braintree_data(self):
             'trial_period': False,
             'name': 'Mozilla',
         }
+
+
+class WebhookVerifyForm(forms.Form):
+    bt_challenge = forms.CharField()
+
+    @property
+    def braintree_data(self):
+        return self.cleaned_data['bt_challenge']
+
+
+class WebhookParseForm(forms.Form):
+    bt_signature = forms.CharField()
+    bt_payload = forms.CharField()
+
+    @property
+    def braintree_data(self):
+        return (
+            self.cleaned_data['bt_signature'],
```

Comment:
this will be a Unicode object but the sample code in the docs show encoding this as ascii before passing it to the API for verification. Does it work without doing that?